service/frontend/workflow_handler.go

```diff
@@ -402,6 +405,23 @@ func (wh *WorkflowHandler) StartWorkflowExecution(ctx context.Context, request *
 		return nil, err
 	}
 
+	for _, callback := range request.GetCompletionCallbacks() {
+		if !wh.config.EnableCallbackAttachment(request.GetNamespace()) {
+			return nil, status.Error(codes.InvalidArgument, "attaching workflow callbacks is disabled for this namespace")
```

Comment:
Does this work with the telemetry interceptor? I think the interceptor checks for the serviceerror.InvalidArgument error type.  Is there any concern using serviceerror.InvalidArgument?

Reply:
I added support in that interceptor.
I think we should start adopting this practice for the following reasons:
- It's the standard way to construct grpc errors
- It avoids sending details as headers where most SDKs don't care about the details
- `status.Errorf` is a useful little helper